### DISCUSSION TITLE
Set tax_id to be fillable

### DISCUSSION THREAD

**Discussion by Ifnir** [getcandy/candy-api#386] [Ideas]
The project I'm workin on is importing from a external Microsoft dynamic service and all the product needs to have the default tax id set, but it's null because tax_id is not fillable.

**Comment by alecritson** [comment]
Hello. I've added this to release [`0.12.5`](https://github.com/getcandy/candy-api/releases/tag/0.12.5) so if you update it should be all good to go for you.
